fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -642,6 +644,9 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = ENABLE_FUNCTION_PUSHDOWN)
     public boolean enableFunctionPushdown = true;
 
+    @VariableMgr.VarAttr(name = ENABLE_COMMON_EXPR_PUSHDOWN, fuzzy = true)
+    public boolean enableCommonExprPushdown = true;
```

Comment:
should add fuzzy code logic in initFuzzyModeVariables method.

Reply:
done